docs/apps.md

```diff
@@ -61,11 +74,21 @@ An example success response to an action with the id `"123"`, would look like:
 }
 ```
 
-> **NOTE:** Uplink broadcasts actions to all connected applications and expects only one application to handle the action. There is a timeout mechanism which on being triggered will send a ***Failed*** response to platform and stop forwarding any *Progress* responses from the connected applications. In order to not trigger this timeout, an application must keep sending timely *Progress* responses(faster than once every 30 seconds) until a final ***Failed*** or ***Completed*** response is made to uplink.
+> **NOTE:** There is a timeout mechanism which on being triggered will send a ***Failed*** response to platform and stop forwarding any *Progress* responses from the connected applications. In order to not trigger this timeout, an application must send a ***Failed*** or ***Completed*** response before the action timeout. Once an action has timedout, a failure response is sent and all it's future responses are dropped. Action timeouts can be configured per action when setting up uplink, as follows:
+> ```
+> [tcpapps.main_app]
+> port = 5555
+> actions = [{ name = "abc", timeout = 300 }] # Allow the connected app to send responses for action abc upto 5 minutes from receive, send a failure response and drop all responses afterwards if not yet completed.
```

Comment:
Can we confirm the docs match uplink behavior?

Reply:
![Screenshot from 2023-06-13 20-05-21](https://github.com/bytebeamio/uplink/assets/18750864/0735d176-ff61-444f-980a-6001b7a7bd9e)
![Screenshot from 2023-06-13 20-05-00](https://github.com/bytebeamio/uplink/assets/18750864/c4f7237a-ff1b-4c13-a121-555478c89da8)